pkg/collector/process.go

```diff
@@ -49,7 +49,8 @@ type CollectingProcess struct {
 	clients map[string]*clientHandler
 	// isEncrypted indicates whether to use TLS/DTLS for communication
 	isEncrypted bool
-	// serverCert and serverKey are for storing encryption info when using TLS/DTLS
+	// caCert, serverCert and serverKey are for storing encryption info when using TLS/DTLS
+	caCert     []byte
```

Comment:
So, we need CACert as a public key to verify or authenticate the owner of ServerCert/ClientCert + ServerKey/ClientKey. Is my understanding correct?

Reply:
Yes.